### DISCUSSION TITLE
Can I access job status from `ACTIONS_RUNNER_HOOK_JOB_COMPLETED` hook script?

### DISCUSSION THREAD

**Discussion by Kesin11** [community/community#28644] [Actions]
Hi, I'm trying to use self-hosted runner's [hook script](https://docs.github.com/en/actions/hosting-your-own-runners/running-scripts-before-or-after-a-job#triggering-the-scripts) for logging about ran jobs on runners.
Each runners are container not VM and deployed to AWS ECS for scaling. Those runner containers will recreate after a job finished with `--ephemeral` option.

In such that architecture, I would like logging and aggregating job data like repository name, job name and **job status** to external storage.
Hook script can access almost job data from runner's [default environemtn variables](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) and webhook event json from [`GITHUB_EVENT_PATH`](https://docs.github.com/en/actions/hosting-your-own-runners/running-scripts-before-or-after-a-job#writing-the-scripts), but can't access job status. As far as I know it exposed by only [job context](https://docs.github.com/en/actions/learn-github-actions/contexts#job-context) and hook script can't access to any context that normally can access inside workflow steps.

My question is, can job completed hook script access to job status?
It maybe possible using `gh run list` or some GitHub API, but these need PAT because hook script can't also access `secrets.GITHUB_TOKEN` context.

**Comment by edo248** [comment]
Hey, did you ever find a solution for this?
